README.md

```diff
@@ -23,22 +23,52 @@ $ python setup.py install # install py-common
 
 **Note:** Before using the py-common in your operating system, you need to make a two-step preparation:
 
+### 1.Configure your cryption tool and signature tool
+
 1. Build executables with sdk-go-common cryption tools. To build these tools, you may need to install **golang** package **sdk-go-common**. For more details please refer to [sdk-go-common](https://github.com/arxanchain/sdk-go-common/tree/master/crypto/tools/README.md)
 
-2. Copy executables **crypto-util** and **sign-util** into your py-common installation path `cryption/utils`.
+2. Copy executables **crypto-util** and **sign-util** into your py-common installation path `cryption/utils`
 
-If you have no idea where your py-common is installed, use the following command to check out.
+If you have no idea where your py-common is installed, use the following command to check out(you need to jump out of your git clone directory before running this command).
```

Comment:
to check it out (you need to leave the `py-common` code repo before running this command).